mParticle-OneTrust/MPKitOneTrust.m

```diff
@@ -62,6 +66,17 @@ - (void)start {
             // Generate consents states for known events
             [self createConsentEvent:consentKey :status];
         }
+        
+        for(NSString *consentKey in [self.venderConsentMapping allKeys]) {
+            // Add consent change observer for all known OneTrust Vendor Events
+            [NSNotificationCenter.defaultCenter addObserver:self selector:@selector(actionConsent:) name:consentKey object:NULL];
```

Comment:
Looks like this observer is never being removed? Nor was the pre-existing one on line 61... Could cause a crash unless we're cleaning this up somewhere I'm not seeing?

Reply:
Also, nitpick, but the `object` parameter should be `nil` rather than `NULL` since it's an object (same with line 61), but they both resolve to 0, so it's more of a style nitpick than a real problem.

Reply:
@einsteinx2 I'm pretty sure removal is unnecessary anymore presuming you want the observer for the whole lifecycle of the app. https://stackoverflow.com/a/40339926/11131375

I'll change those nils 👍